Source/ACE.Server/WorldObjects/Player_Move.cs

```diff
@@ -50,7 +50,7 @@ public void CreateMoveToChain(WorldObject target, Action<bool> callback, float?
             // fix bug in magic combat mode after walking to target,
             // crouch animation steps out of range
             if (useRadius == null)
-                useRadius = target.UseRadius ?? 0.6f;
+                useRadius = target.UseRadius ?? 1f;
```

Comment:
this is a client hardcoded value. this should not be changed arbitrarily.

Reply:
ok, can you provide the source point for it? 

Issue is at less than 1f, items on desks, tables, and similar furniture cannot be reachable without getting on top of them.

Reply:
LODWORD(this->distance_to_object) = 1058642330;
public static readonly float Default_DistanceToObject = 0.6f;

this can also be seen in all of the pcaps where it sends 0.6f for the default DistanceToObject for almost everything in the game.

Reply:
very well. How do we know that UseRadius == Default_DistanceToObject, is it at all possible they are different?

Reply:
just examine the pcaps for any object. it sends the UseRadius for DistanceToObject in the MoveToObject MovementParameters